quiz.js

```diff
@@ -47,10 +47,13 @@ const questions = [
 
   // show next question
   let currentQuestion = 0;
+  let correctAnswers = 0;
+  let currentAnswer = undefined;
```

Comment:
correctAnswers  liczymy tu, bo potem będziemy odczytywać to z różnych miejsc ale currentAnswer wystarczy, że zdefiniujemy tylko w funkcji verify bo tylko tam będzie potrzebna

Reply:
ok